e2e/install-conjur-no-pv.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-helm install \
+helm install conjur-e2e \
```

Comment:
what does this change do? I looked at my local `helm install --help` to try to figure it out, but it's not 100% clear to me

Reply:
with helm v3+, you have to have the name of the deployment passed in:
```
helm version
version.BuildInfo{Version:"v3.0.1", GitCommit:"7c22ef9ce89e0ebeb7125ba2ebf7d421f3e82ffa", GitTreeState:"clean", GoVersion:"go1.13.4"}

$ helm install
Error: "helm install" requires at least 1 argument

Usage:  helm install [NAME] [CHART] [flags]
```

Reply:
Before we merge can you please
- move the maintenance info to CONTRIBUTING.md instead of the README (including release instructions)
- add a comment on the required helm version to the release instructions